docs/static/setup.sh

```diff
@@ -11,7 +11,7 @@ title="ZMK Config Setup:"
 # TODO: Check for user.name and user.email git configs being set
 
 prompt="Pick an MCU board:"
-options=("nice!nano" "QMK Proton-C")
+options=("nice!nano" "QMK Proton-C" "bluemicro52840_v1")
```

Comment:
This should be the "friendly name", e.g.:

```suggestion
options=("nice!nano" "QMK Proton-C" "BlueMicro52840 (v1)")
```

Also, you need to add a selection in the `select` block below that will set `board="bluemicro52840_v1"`.